com.unity.shadergraph/Editor/Data/Util/GraphUtil.cs

```diff
@@ -280,7 +280,7 @@ static void Visit(List<AbstractMaterialNode> outputList, Dictionary<string, Abst
         internal static string SanitizeName(IEnumerable<string> existingNames, string duplicateFormat, string name)
         {
             //.shader files are not cool with " in the middle of a property name (eg.  Vector1_81B203C2("fo"o"o", Float) = 0)
```

Comment:
Comment should be changed to reflect actual code. Would be useful to link to .NET API regex page to be clear what "\W" covers. Should we also document this?

Reply:
do you want a full API link, or is specifying `\w, \W: ANY ONE word/non-word character. For ASCII, word characters are [a-zA-Z0-9_]` enough? so in this case, it's replacing any non word character with _

Reply:
I think the second idea is great.

Reply:
Actually, this doesn't apply here any more since it's a generic regex. Comment applies to default param, so we can leave it. Closing.